core/src/test/scala/org/apache/spark/storage/BlockManagerSuite.scala

```diff
@@ -1415,6 +1415,79 @@ class BlockManagerSuite extends SparkFunSuite with Matchers with BeforeAndAfterE
       super.fetchBlockSync(host, port, execId, blockId)
     }
   }
+
+  def testGetOrElseUpdateForLargeBlock(storageLevel: StorageLevel) {
```

Comment:
yes, currently working on:
1.  parameterizing `DiskStore` and `DiskStoreSuite`
2. revert the tests in `BlockManagerSuite` 
3. revert the 6gb  change in sbt